test/pools/LinearLiquidityPool/BaseMulticon.sol

```diff
@@ -0,0 +1,125 @@
+pragma solidity >=0.6.0;
+
+import "truffle/Assert.sol";
+import "truffle/DeployedAddresses.sol";
+import "../../../contracts/deployment/Deployer.sol";
+import "../../../contracts/finance/OptionsExchange.sol";
+import "../../../contracts/finance/OptionToken.sol";
+import "../../../contracts/pools/LinearLiquidityPool.sol";
+import "../../../contracts/governance/ProtocolSettings.sol";
+import "../../common/actors/PoolTrader.sol";
+import "../../common/actors/OptionsTrader.sol";
+import "../../common/mock/ERC20Mock.sol";
+import "../../common/mock/EthFeedMock.sol";
+import "../../common/mock/TimeProviderMock.sol";
+
+contract StablecoinMock is ERC20Mock{
```

Comment:
According to solidity style guide there should be a spacing after the contract name: `ERC20Mock {`
Please review all other occurencies of this in the PR code.

Reply:
For reference: https://docs.soliditylang.org/en/v0.5.3/style-guide.html